src/components/Modals/UploadButtonGroup.jsx

```diff
@@ -53,24 +107,28 @@ const UploadButtonGroup = ({ file, setFile }) => {
           accept=".pdf, .docx, .doc, .txt, .rtf, .gif, .png, .jpeg, .jpg, .webp"
         />
       </Button>
-      {isCustomMediumScreen && (
+
+      {hasCamera && (
         <Button
           variant={file ? 'outlined' : 'contained'}
           component="label"
           color="primary"
-          id="upload-doctype"
-          name="uploadDoctype"
-          onChange={(e) => setFile(e.target.files[0])}
+          onClick={() => setShowWebcamModal(true)}
           fullWidth
-          required
           startIcon={<PhotoCameraIcon />}
+          sx={{ borderRadius: '20px', marginLeft: '8px' }}
         >
-          Capture image
-          <input type="file" hidden accept="image/*" capture="environment" />
+          Use Webcam
         </Button>
       )}
```

Comment:
There really should be two buttons, but one only renders depending on condition. So instead of:

```
{hasCamera && <WebCam Button>}
```

It really should be:

```
{hasCamera ? <WebCam Button> : <Front-facing Cam Button>}
```

You want to keep the old version to cover both cases.


https://github.com/codeforpdx/PASS/assets/14917816/d51fba7d-2499-4d1d-981f-23807c386db6